docs/en/operations/system-tables/dictionaries.md

```diff
@@ -21,6 +21,7 @@ Columns:
 -   `bytes_allocated` ([UInt64](../../sql-reference/data-types/int-uint.md#uint-ranges)) — Amount of RAM allocated for the dictionary.
 -   `query_count` ([UInt64](../../sql-reference/data-types/int-uint.md#uint-ranges)) — Number of queries since the dictionary was loaded or since the last successful reboot.
 -   `hit_rate` ([Float64](../../sql-reference/data-types/float.md)) — For cache dictionaries, the percentage of uses for which the value was in the cache.
+-   `found_rate` ([Float64](../../sql-reference/data-types/float.md)) — The percentage of uses for which the value was found.
```

Comment:
Could not we utilize hit rate, without introducing new attribute, before hit_rate was only defined for cache dictionaries ?

Reply:
I though about this, but I would prefer not to, since for cache dictionaries it is also interesting and if hit_rate will be reused then for cache dictionaries this cannot be implemented.

Reply:
I am not sure but what difference will be with hit_rate and found_rate for cache dictionaries ?

Reply:
hit_rate is about what data is in cache, found_rate is about what data is in the source (+cache of course)